### PR TITLE
Bug 2007522: No new local-storage-operator-metadata-container is buil…

### PR DESCRIPTION
**Pull request by dobsonj**
https://bugzilla.redhat.com/show_bug.cgi?id=2007522
/cc @openshift/storage

### CODE REVIEW

File: config/manifests/4.10/local-storage-operator.v4.9.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -136,7 +136,7 @@ spec:
     name: Red Hat
   labels:
     alm-owner-metering: local-storage-operator
-    alm-status-descriptors: local-storage-operator.v4.9.0
+    alm-status-descriptors: local-storage-operator.v4.10.0
   selector:
     matchLabels:
       alm-owner-metering: local-storage-operator
```
Comment (gnufied) [inline config/manifests/4.10/local-storage-operator.v4.9.0.clusterserviceversion.yaml:142]: Forgot to rename the CSV file?

Comment (dobsonj) [reply config/manifests/4.10/local-storage-operator.v4.9.0.clusterserviceversion.yaml]: Good catch, updated.

**Pull request merged**